moabb/paradigms/motor_imagery.py

```diff
@@ -68,15 +75,16 @@ def process_raw(self, raw, dataset):
                                include=channels)
 
         # get event id
-        if self.events is None:
-            event_id = dataset.event_id
-        else:
-            event_id = {ev: dataset.event_id[ev] for ev in self.events}
+        event_id = self.used_events(dataset)
 
         # pick events, based on event_id
         try:
             events = mne.pick_events(events, include=list(event_id.values()))
-        except RuntimeError:
+            # not all runs have all events. Reduce event dict to only occurring events
+            events_in_run = np.unique(events[:, 2])
```

Comment:
I think this is not necessary if we set the parameter `on_missing='ignore'` in the mne.Epochs call

Reply:
Will check this out